tests/integration/conftest.py

```diff
@@ -1,40 +1,41 @@
-import shutil
-from contextlib import contextmanager
+import tempfile
+from pathlib import Path
 
+import ape
 import pytest
-from ape.utils import create_tempdir
 from click.testing import CliRunner
 
-from ape_safe._cli import cli as CLI
+from ape_safe._cli import cli as ape_safe_cli
 
 
 @pytest.fixture
-def runner():
-    return CliRunner()
+def safe_container():
+    return ape.accounts.containers["safe"]
+
+
+# NOTE: Every test gets a different data folder
+@pytest.fixture(scope="function", autouse=True)
+def patch_data_folder(monkeypatch, safe_container):
+    with tempfile.TemporaryDirectory() as data_folder_override:
```

Comment:
`ape.utils.create_tempdir` is nice because it fixes a really annoying bug for when you don't resolve the path _and_ it already returns a Path object

Reply:
922a997